google/api_core/exceptions.py

```diff
@@ -461,6 +480,41 @@ def _is_informative_grpc_error(rpc_exc):
     return hasattr(rpc_exc, "code") and hasattr(rpc_exc, "details")
 
 
+def _parse_grpc_error_details(rpc_exc):
+    if not rpc_status:
+        return []
+    if not isinstance(rpc_exc, grpc.Call):
+        return []
+    status = rpc_status.from_call(rpc_exc)
+    if not status:
+        return []
+    possible_errors = [
+        error_details_pb2.BadRequest,
+        error_details_pb2.PreconditionFailure,
+        error_details_pb2.QuotaFailure,
+        error_details_pb2.ErrorInfo,
+        error_details_pb2.RetryInfo,
+        error_details_pb2.ResourceInfo,
+        error_details_pb2.RequestInfo,
+        error_details_pb2.DebugInfo,
+        error_details_pb2.Help,
+        error_details_pb2.LocalizedMessage,
+    ]
```

Comment:
Can we compute this list coditionally as `[]` if `details_pb2` is not importable?